conductor/blueprints/authentication/controllers.py

```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function
```

Comment:
For Python 2/3 code with we are writing here, I strongly recommend that we always start files like:

```
# -*- coding: utf-8 -*-
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
from __future__ import unicode_literals
```

Which prevents many potential surprises, and provides API consistency.